packages/frontend-2/components/error/page/GenericErrorBlock.vue

```diff
@@ -3,18 +3,24 @@
   <div class="flex flex-col items-center space-y-8">
     <!-- <ErrorPageProjectInviteBanner /> -->
     <h1 class="h1 font-medium">Error {{ error.statusCode || 500 }}</h1>
-    <h2 class="h3 text-foreground-2 mx-4 break-words max-w-full">
-      {{ error.message }}
-    </h2>
+    <div class="flex flex-col items-center space-y-1">
+      <h2 class="h2 text-foreground-2 mx-4 break-words max-w-full">
+        {{ error.message }}
+      </h2>
+      <div class="text-foreground-2 text-xs">Reference #{{ reqId }}</div>
```

Comment:
Could you use the new text class here? `text-body-xs` or `text-body-2xs` probably :D

Reply:
@Mikehrn I wanted it to be even smaller than text-body-2xs, really want it to be as tiny as possible 🤔

Reply:
@fabis94 There is also -3xs, if that's still too big lets go with this then :)